### PR TITLE
[Chore #108330926]make userprofie app unique

### PR DESCRIPTION
**Pull request by andela-sjames**

### CODE REVIEW

**Comment by andela-sjames** [comment]
Userprofile migration complete

**Comment by andela-sjames** [comment]
This branch can be merged

File: troupon/userprofile/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from userprofile.views import Userprofileview
+
+urlpatterns = [
+url(r'^user/(?P<username>\w+)$', Userprofileview.as_view(), name = 'userprofile'),
```
Comment (andela-cnnadi) [inline troupon/userprofile/urls.py]: *Code Quality: Nitpick*

Not sure we still need `/user` in the URL

File: troupon/userprofile/templates/userprofile/profile.html [hunk 2]
```diff
@@ -38,7 +38,7 @@
                 <p> Interest:  {{ profile.interest }} </p>
             </div>
 
-            <form action="/account/profile/user/{{ request.user.username }}" method="POST" id="profile-section"><input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
+            <form action="/userprofile/user/{{ request.user.username }}" method="POST" id="profile-section"><input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
```
Comment (andela-cnnadi) [inline troupon/userprofile/templates/userprofile/profile.html]: **Code Quality: Nitpick**

You should use `url` here. This makes it easier to reference URLs especially if they are modified eventually.

File: troupon/userprofile/urls.py (hunk 1 above)
Comment (andela-cnnadi) [inline troupon/userprofile/urls.py]: You should also indent the code properly. See below:
````python
urlpatterns = [
    url(r'^user/(?P<username>\w+)$', Userprofileview.as_view(), name = 'userprofile'),
]
```

**Comment by andela-sjames** [comment]
This branch can be merged

**Pull request merged**